src/double_imanip.h

```diff
@@ -21,52 +21,59 @@
 /**
    \file  double_imanip.h
 
-   IO manipulation classes for parsing nan and inf.
+   Fast double input stream with support for parsing nan and inf.
 
-   Not all compilers do support parsing of nan and inf.
-   The code below is taken (and slightly modified) from the discussion at:
-   http://stackoverflow.com/questions/11420263/is-it-possible-to-read-infinity-or-nan-values-using-input-streams
-
-   \author Oliver Lemke
-   \date 2021-06-03
+   \author Oliver Lemke, Richard Larsson
+   \date 2022-05-23
 */
 
 #ifndef double_imanip_h
 #define double_imanip_h
 
 #include <fstream>
 
```

Comment:
I wonder why `<limits>` is included in clang but not GCC, nevertheless it looks good bar GCC's lack of inclusions